blackbox/test_docs.py

```diff
@@ -38,6 +38,12 @@
 from crate.client import connect
 
 
+FILE_NAME_FILTER = os.environ.get('FILE_NAME_FILTER')
+if FILE_NAME_FILTER:
+    print("Applying file name filter: {}".format(FILE_NAME_FILTER))
+is_target_file_name = lambda item: not FILE_NAME_FILTER or (item and FILE_NAME_FILTER in item)
```

Comment:
I think pep8 /pycodestyle would complain about this as lambda's shouldn't be assigned to variables, but `def ...` should be used instead.

Reply:
Correct, thank you for that, I have pushed a commit to fix it.